mrtt-ui/src/views/Auth/LoginForm.js

```diff
@@ -0,0 +1,108 @@
+import { Controller, useForm } from 'react-hook-form'
+import { toast } from 'react-toastify'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
+import axios from 'axios'
+
+import { ButtonCancel, ButtonSubmit } from '../../styles/buttons'
+import { ButtonContainer, RowFlexEnd } from '../../styles/containers'
+import { ErrorText } from '../../styles/typography'
+import { Form, MainFormDiv, SectionFormTitle } from '../../styles/forms'
+import { FormLabel, TextField } from '@mui/material'
+import language from '../../language'
+import LoadingIndicator from '../../components/LoadingIndicator'
+import getBaseUrl from '../../library/auth/getBaseUrl'
+
+const validationSchema = yup.object({
+  email: yup.string().required('Email required'),
+  password: yup.string().required('Password required')
+})
+
+const formDefaultValues = { email: '', password: '' }
+
+const LoginForm = () => {
+  const [isLoading] = useState(false)
+  const [isSubmitError, setIsSubmitError] = useState(false)
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const navigate = useNavigate()
+
+  const authUrl = `${getBaseUrl(process.env.REACT_APP_API_URL)}/users/sign_in`
+
+  const {
+    control: formControl,
+    handleSubmit: validateInputs,
+    formState: { errors }
+  } = useForm({ resolver: yupResolver(validationSchema), defaultValues: formDefaultValues })
+
+  const options = {
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json'
+    }
+  }
+
+  const signUp = (formData) => {
+    axios
+      .post(authUrl, { user: formData }, options)
+      .then(({ data }) => {
+        setIsSubmitting(false)
+        toast.success(data.message)
+
+        if (data.token) {
+          localStorage.setItem('token', data.token)
+          // TODO: navigate to another page
```

Comment:
Lets avoid using the code base for project management. Can you make a ticket for this comment in our issue tracker instead?

Reply:
Agree. I have another PR ready that addresses this.